src/styles/issues.scss

```diff
@@ -98,6 +98,8 @@
     border-radius: 4px;
     box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.2);
     font-family: $font-1;
+    max-height: -webkit-fill-available;
```

Comment:
Using -webkit in front of a css value means that it will only be used in webkit based browsers. The best practise is to use the fill-available value and depend on tools like [autoprefixed](https://github.com/postcss/autoprefixer) to provide vendor specific tags.

The issue with the fill-available tag is that it isn't well supported https://caniuse.com/#search=fill-available , could you replace it with a vh value?